pkgs/by-name/um/umu-launcher/package.nix

```diff
@@ -2,12 +2,15 @@
   buildFHSEnv,
   lib,
   umu-launcher-unwrapped,
+  extraPkgs ? pkgs: [ ],
+  extraLibraries ? pkgs: [ ],
 }:
 buildFHSEnv {
   pname = "umu-launcher";
   inherit (umu-launcher-unwrapped) version meta;
 
-  targetPkgs = pkgs: [ pkgs.umu-launcher-unwrapped ];
+  targetPkgs = pkgs: [ pkgs.umu-launcher-unwrapped ] ++ extraPkgs pkgs;
+  multiPkgs = pkgs: extraLibraries pkgs;
```

Comment:
Could end-users already do this with `overrideAttrs`?

```nix
umu-launcher.overrideAttrs (old: {
  targetPkgs = pkgs: old.targetPkgs pkgs ++ extraPkgs pkgs;
  multiPkgs = extraLibraries;
})
```

I guess doing it _safely_ could get ugly:

```nix
umu-launcher.overrideAttrs (old: {
  targetPkgs = pkgs: (old.targetPkgs or (_: [ ])) pkgs ++ extraPkgs pkgs;
  multiPkgs = pkgs: (old.multiPkgs or (_: [ ])) pkgs ++ extraLibraries pkgs;
})
```

Reply:
Answered my own question: https://github.com/NixOS/nixpkgs/pull/375423#issuecomment-2605310343

TL;DR: no, you can't use `overrideAttrs`.